packages/xstate-wallet/src/store/memory-store.ts

```diff
@@ -106,9 +106,8 @@ export class MemoryStore implements Store {
   public async createChannel(
     participants: Participant[],
     challengeDuration: BigNumber,
-    stateVars: StateVariables,
```

Comment:
What's the logic here? Don't you always know what the first state will be at the time you create the channel?

Reply:
It just became an unused variable.